settings.gradle.kts

```diff
@@ -1,19 +1,30 @@
+import java.util.Properties
+
+
 pluginManagement {
     repositories {
         gradlePluginPortal()
         google()
         mavenCentral()
-        jcenter()
-        maven { url "https://jitpack.io" }
+        maven {
+            setUrl("https://jitpack.io")
+        }
     }
+    plugins {
+        kotlin("jvm") version "2.1.20"
+    }
```

Comment:
Why do we need this plugin?
It seems to be needed for JVM projects, not Android
Plugins are usually not specified in the settings.gralde